skills/declarative/Calendar/Calendar/dialogs/AuthenticationDialog/AuthenticationDialog.dialog

```diff
@@ -3,7 +3,8 @@
   "$designer": {
     "id": "aZc9SE",
     "name": "AuthenticationDialog",
-    "description": "Authenticates user through AAD and accesses their MS Graph user profile and mail box settings to capture their name and time zone."
+    "description": "Authenticates user through AAD and accesses their MS Graph user profile and mail box settings to capture their name and time zone.",
```

Comment:
I ended up duplicating the same comment on the dialog level and in the BeginDialog trigger so that if the user clicks in either place, they will be able to see what the dialog does. The only place this is different is in the CreateEvent and UpdateEvent dialogs where I modify the BeginDialog trigger to include a bit more info about how the flow works differently in those dialogs.